http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -493,12 +504,17 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
             ).map(b => (imageType, b))
           )
       }
-      // wait for 10 minutes for tools to start up before time out.
+      // wait for tools to start up before time out. Use the timeout value if available
```

Comment:
This is really where the meat of this work is. If the timeout is not None then use that, else use the default